addons/l10n_ar_website_sale/demo/website_demo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="default_website_ri" model="website">
+        <field name="name">(AR) Responsable Inscripto Website</field>
+        <field name="domain"></field>
+        <field name="company_id" ref="l10n_ar.company_ri"/>
+        <field name="logo" type="base64" file="l10n_ar_website_sale/static/description/icon.png"/>
+        <field name="domain" model="ir.config_parameter" eval="obj().env['ir.config_parameter'].sudo().get_param('web.base.url')"/>
+    </record>
+    <function model="payment.acquirer" name="copy">
+        <value eval="[ref('payment.payment_acquirer_transfer')]"/>
+        <value eval="{'company_id': ref('l10n_ar.company_ri'),'state': 'enabled'}"/>
+    </function>
+    <record id="list0" model="product.pricelist">
+        <field name="name">AR Public Pricelist</field>
+        <field name="selectable" eval="False"/>
+        <field name="website_id" ref="l10n_ar_website_sale.default_website_ri"/>
```

Comment:
Hi @jco-odoo 

The purpose will be to let the user to fulfill a complete ecommerce checkout as it works on an Argentinean company:
* an Argentinean website in order to properly see the fields in the shop checkout form
* payment acquirer same as Transfer Wire just to let the user to complete the checkout process
* an pricelist to be able to see amounts in pesos argentinos.
* product.product write function just to ensure that all the products in the website has at less one tax of type vat that is required for the invoice lines of Argentinean invoices

Reply:
@jco-odoo what do you think about this? we can keep this demo data?

Reply:
@jco-odoo I think by default there is a **Default ARS pricelist** related to **My Website** (see image bellow). But the one that is showing in the **(AR) Responsable Inscripto Website** is the **Public Pricelist** which has higher priority and is represent in USD. 

To ensure that the website show a pricelist in ARS we are forcing to create a pricelist in ARS that is related directly to the website, this way will show this pricelist and not the Public Pricelist by default. If you prefer, maybe instead of create a new pricelist, we can find **Default ARS pricelist** and add it the **(AR) Responsable Inscripto Website**?

![image](https://user-images.githubusercontent.com/7593953/88290468-37844a00-cccd-11ea-809f-9a46484c23b8.png)